test/check-machines-nics

```diff
@@ -139,6 +140,26 @@ class TestMachinesNICs(VirtualMachinesCase):
         b.wait_not_present(".pf-c-modal-box")
         b.wait_not_present("#vm-subVmTest1-network-1-mac")
 
+        # Detach NIC when the VM is shutoff, Also check this issue: https://bugzilla.redhat.com/show_bug.cgi?id=1791543
+        self.performAction("subVmTest1", "off")
+        # Attach since no NIC now
+        m.execute("virsh attach-interface --domain subVmTest1 --type network --source default --model virtio --mac 52:54:00:4b:73:5f --current")
+        m.execute("virsh attach-interface --domain subVmTest1 --type bridge --source virbr0 --model virtio --mac 52:54:00:4b:73:6f --current")
+        m.execute("virsh attach-interface --domain subVmTest1 --type network --source default --model virtio --mac 52:54:00:4b:73:4f --current")
+        # Refresh for getting new added NIC
+        b.reload()
+        b.enter_page('/machines')
+        b.wait_in_text("body", "Virtual machines")
+        # Check NIC MAC addresses
+        b.wait_text("#vm-subVmTest1-network-1-mac", "52:54:00:4b:73:5f")
+        b.wait_text("#vm-subVmTest1-network-2-mac", "52:54:00:4b:73:6f")
+        b.wait_text("#vm-subVmTest1-network-3-mac", "52:54:00:4b:73:4f")
+        # Detach
+        self.deleteIface(2)
+        # Make sure popup dialog disappeared
+        b.wait_not_present(".pf-c-modal-box")
+        b.wait_not_present("#vm-subVmTest1-network-2-mac")
```

Comment:
I think this will fail, as the indexing is done according to the position in the list. So deleting the second element, will make the third sedond.

Reply:
OK, re-push. And I found a weird thing. Sometimes, after attaching three network interfaces, the VM disappeared and there is also a weird log in libvirtd(which is "internal error: End of file from qemu monitor"). I think that is the reason why the cases is pass during my last run in my laptop. Do you have any ideas about this libvirt behavior?

Reply:
Hey, sorry - just saw that. I have seen this error as well in the CI a few times, but not locally. If you can conistently reproduce it, please share the steps and we can debug it together.

Reply:
I’m trying to reproduce that, but it seems impossible reproduced manually, so I'm trying to reproduce it by a script, Once reproduced, I will leave a comment

Reply:
I am failed on reproducing that. Also it seems that it could not be reproduced every times, when running verify test( Probably reproduce 3 times in 5 times). Then I try to search suspicious points in the new case. I think the most suspicious point is "--current" since it will let the config take effect directly(get from help information of "virsh attach-interface").So I change "--current" to "--config". Then, I run the case five times, and they are all pass. So let's re-run  jobs for making sure the modification. Also, do you have another ideas?